reservation/test_appointment_model.py

```diff
@@ -68,10 +68,12 @@ def test_del_appointment(self, make_appointment):
         appointment.delete()
         assert appointment not in TypeOfJob.objects.all()
 
-    def test_get_clients_by_professional(self, persisted_appointment_pool, professional):
-        assert persisted_appointment_pool == list(Appointment.get_clients(professional_id=professional))
-
-    def test_get_client_list_on_certain_day_by_professional_and_date(self, persisted_appointment_pool, professional):
+    def test_get_appointment_list_after_current_day_by_professional(self, persisted_appointment_pool, professional):
         assert persisted_appointment_pool == \
-               list(Appointment.get_client_list_on_certain_day(professional_id=professional,
-                                                               date=(current_datetime + timedelta(days=5)).date()))
+               list(Appointment.get_appointments_list_after_current_day(profile_id=professional.professional_id,
+                                                                        type_of_user=True))
+
+    def test_get_appointment_list_after_current_day_by_client(self, persisted_appointment_pool, demo_client):
+        assert persisted_appointment_pool[1] == \
```

Comment:
Please consult with @guybeckenstein how to use parameterized

Reply:
[see lines 38-42 in `landing/tests/test_base_navigation_bar.py`](https://github.com/redhat-beyond/Pro4U/pull/125/files#diff-548f1fdd5149d72034d5ab17fdfa3ebeaf716a4ee544f61c2333d0236dd1a945)

Reply:
Done @Yarboa